elements/bulbs-video/bulbs-video.js

```diff
@@ -47,13 +79,15 @@ Object.assign(BulbsVideo, {
     controller: ControllerField,
   },
   propTypes: {
-    autoplay: PropTypes.bool,
-    autoplayNext: PropTypes.bool,
-    loop: PropTypes.bool,
-    mute: PropTypes.bool,
+    autoplay: PropTypes.string,
+    autoplayNext: PropTypes.string,
+    noEndcard: PropTypes.string,
     src: PropTypes.string.isRequired,
     twitterHandle: PropTypes.isRequired,
   },
 });
 
 registerReactElement('bulbs-video', BulbsVideo);
+
+import './elements/meta';
+import './elements/summary';
```

Comment:
What's the distinction here between elements and components? Is one a function and the other a class?